stl/inc/__msvc_ostream.hpp

```diff
@@ -0,0 +1,1043 @@
+// __msvc_ostream.hpp internal header
+
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef __MSVC_CXX_OSTREAM_HPP

```

Comment:
The CXX here doesn't follow our usual pattern for idempotency guards.

It looks like this was copy-pasted from `__msvc_cxx_stdatomic.hpp`'s macro `__MSVC_CXX_STDATOMIC_HPP`, where it's actually part of the filename.